roles/update_status_html/tasks/main.yml

```diff
@@ -1,7 +1,11 @@
 ---
 # builds that status.html file that shows the deployed versions
 
-- name: get git tag
+#
+# get versions
+#
+
+- name: get local git tag
```

Comment:
Is this information about the repo in ~/git/migrid/ actually usefull anymore when the ansible-docker-migrid is used as a collection?

Reply:
Those two clauses are set with 
`when: migrid_update_status_deploy_version is defined`
I will still use it to know what version of my private git with all the variables are used during deploy. I want to know how it was deployed from here, not just what version of migrid, and what version of ansible-docker-migrid.